src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -1,49 +1,48 @@
 package org.broadinstitute.dsde.firecloud.model
 
-import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
-
-import scala.annotation.meta.field
+import org.broadinstitute.dsde.firecloud.core.AgoraPermissionHandler
 
 object MethodRepository {
 
-  @ApiModel(value = "Configuration")
   case class Configuration(
-                            @(ApiModelProperty@field)(dataType = "string", value = "Namespace")
-                            namespace: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Name ")
-                            name: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Snapshot Id")
-                            snapshotId: Option[Int] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Synopsis")
-                            synopsis: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Documentation")
-                            documentation: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Owner")
-                            owner: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Payload")
-                            payload: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Excluded Field")
-                            excludedField: Option[String] = None,
-                            @(ApiModelProperty@field)(dataType = "string", value = "Included Field")
-                            includedField: Option[String] = None)
+    namespace: Option[String] = None,
+    name: Option[String] = None,
+    snapshotId: Option[Int] = None,
+    synopsis: Option[String] = None,
+    documentation: Option[String] = None,
+    owner: Option[String] = None,
+    payload: Option[String] = None,
+    excludedField: Option[String] = None,
+    includedField: Option[String] = None
+  )
```

Comment:
Re : "I think because you squashed".....I think so, but it squashing supposed to do that?  I guess if only a single name can go onto a given single commit it makes sense